README.md

```diff
@@ -25,36 +26,36 @@ which are not actively developed anymore.
   - Currently only python has bindings, located in `bindings/py`
 
 
-This repository contains the C++ source code for the Numenta Platform for 
-Intelligent Computing ([NuPIC](http://numenta.org/nupic.html)). 
-It will eventually contain all algorithms for NuPIC, but is currently in a transition period. 
+This repository contains the C++ source code for the Numenta Platform for
+Intelligent Computing ([NuPIC](http://numenta.org/nupic.html)).
+It will eventually contain all algorithms for NuPIC, but is currently in a transition period.
 
-\*) Nupic API compatability: The objective is to stay as close as possible to the [Nupic API Docs](http://nupic.docs.numenta.org/stable/api/index.html) 
-with the aim that we don't break `.py` code written against the numenta's nupic.core extension library if they were to be 
+\*) Nupic API compatability: The objective is to stay as close as possible to the [Nupic API Docs](http://nupic.docs.numenta.org/stable/api/index.html)
+with the aim that we don't break `.py` code written against the numenta's nupic.core extension library if they were to be
 ran against this extention library. If you are porting your code to this codebase, please review [API Changelog](API_CHANGELOG.md).
 
-### New Features 
+### New Features
```

Comment:
ok now, but for the future please fix your editor to avoid these "automated" whitespace changes. It makes the review more difficult.